.github/workflows/publish.yml

```diff
@@ -39,15 +38,45 @@ jobs:
       - name: Run Postinstall Script
         if: steps.cache.outputs.cache-hit == 'true'
         run: npm run postinstall
+      - name: Check modified files
+        id: modified-files
+        uses: dorny/paths-filter@v2.9.2
+        with:
+          filters: |
+            core:
+              - 'libs/core/**'
       - name: Configure git
         run: |
           git config user.name "Kirby Bot"
           git config user.email "<>"
+      - name: Create npm config
+        if: steps.modified-files.outputs.core == 'true'
+        run:  |
+          cd libs/core 
+          echo '//registry.npmjs.org/:_authToken=${{secrets.NPM_PUBLISH_TOKEN}}' > ~/.npmrc
+      - name: Bump core package and update version in designsystem
+        id: bump-core
+        if: steps.modified-files.outputs.core == 'true'
+        run: | # patch version -> write without 'v' to variable -> update core package dependency in designsystem -> commit patch manually due to https://github.com/npm/cli/issues/2010#issuecomment-863288622 
```

Comment:
Is comment left here on purpose? 
If it is consider plopping it onto a line of its own - it seems like it has been left here by mistake :)

Reply:
I changed this to a comment on this PR, and linked that instead!

Reply:
cool 👌